roles/kubernetes/master/defaults/main.yml

```diff
@@ -56,6 +56,20 @@ kube_apiserver_admission_control:
       {%- endif -%}
   - ResourceQuota
 
+# 1.10+ admission plugins
+kube_apiserver_enable_admission_plugins:
```

Comment:
Actually there's no need to populate this variable regarding the default On plugins (https://github.com/hzxuzhonghu/kubernetes/blob/a00e3748d4dac523d893cd67cfd73f41d4890b0f/pkg/kubeapiserver/options/plugins.go#L132)

Reply:
Even better!